packages/react-components/react-storybook-addon/src/components/DirectionSwitch.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { IconButton } from '@storybook/components';
+
+import { DIR_ID } from '../constants';
+import { useGlobals } from '../hooks';
+
+const monospaceStyle = {
```

Comment:
for this use case I guess this is ok but going forward we should use storybook apis (emotion)

```
import { styled } from '@storybook/theming';
...
```

see https://storybook.js.org/docs/addons/addon-knowledge-base#style-your-addon

Reply:
I thought this implementation was Storybook 8 exclusive. I updated the code to use Emotion.

Reply:
yeah navigating in storybook related things is challenging . ty!